### PR TITLE
Front/119/reports

### PR DESCRIPTION
**Pull request by hilvitzs**
### This pull request changes...
This adds the initial reports page which should match the mock below.
Screen Shot 2020-12-09 at 10 41 05 AM

There should be a route for `Reports` with a corresponding navigation tab that is highlighted when on any variation of `/reports/*`.

There should be a page title of 'TANF Reports' and the browser tab should display that name
There should be a `Select` dropdown that allows the user to select either the year `2020` or `2021` for the ATO. When you make a selection it should update the `p` tag underneath the select with the selected year.
The label for the `Select` dropdown should read "Fiscal Year (October - September)"
There should be a button to `Begin Report` and upon click of this button the user should be routed to `/reports/[the user's selected year]/upload'. There won't be anything on that page yet but that work will be completed in #429 .

This addresses issue #119 

## TO TEST

*List the steps to test the PR*

Log in to the application and you will be directed to `/edit-profile`.
There should be a navigation tab that reads `Reports` - click on that and you should be rerouted to `/reports` where you will see the new page.

You should be able to choose between 2020 and 2021. When you have chosen the year, it should update the `p` tag below with the selected year. When you have made a selection, click on `Begin Report` and you should be rerouted to `/reports/[the year you selected]/upload`.

### This pull request is ready to merge when...

- [ ] Meets acceptance criteria for the issue 
- [ ] Code is reviewed by someone other than the original author
- [ ] Code is tested 
- [x] Experience is approved by UX  
- [ ] Meets [a11y checklist](https://github.com/HHS/TANF-app/blob/main/docs/a11y/how-18f-will-test-a11y.md)
- [ ] Meets [Raft's Manual QASP Checklist](https://teams.microsoft.com/l/file/B5285301-13CA-46FD-A723-AFFD578C224A?tenantId=d58addea-5053-4a80-8499-ba4d944910df&fileType=docx&objectUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA%2FShared%20Documents%2FGeneral%2FManual%20Testing%20Practices_Issue173.docx&baseUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA&serviceName=teams&threadId=19:f769bbcb029f4f02b55ae7fad90e310d@thread.skype&groupId=41f194a6-c1d3-4680-933e-c8ee7d17e287):lock: (only applicable for PR to main HHS)
- [ ] Documentation is updated 
  - OpenAPI  
  - Readme
  - Entity Relationship Diagram (ERD)

### CODE REVIEW

File: tdrs-frontend/src/components/NavItem/NavItem.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+/**
+ *
+ * @param {string} pathname - The full current url path.
```
Comment (alexsoble) [inline tdrs-frontend/src/components/NavItem/NavItem.jsx:6]: 👍 👍 JSDoc-style documentation!

File: tdrs-frontend/src/components/Reports/Reports.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import Button from '../Button'
+import { history } from '../../configureStore'
+import { setYear } from '../../actions/upload'
+
+/**
+ * @param {string} selectedYear = The year that the user has chosen from the
```
Comment (alexsoble) [inline tdrs-frontend/src/components/Reports/Reports.jsx:8]: 👍 👍 JSDoc-style documentation!

File: tdrs-frontend/src/components/NavItem/NavItem.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+ * on the navigation tab.
+ * @param {string} href - The path for where the user should
+ * be redirect on click.
+ */
+function NavItem({ pathname, tabTitle, href }) {
+  return (
+    <li className="usa-nav__primary-item">
+      <a
+        href={href}
+        key="welcome"
```
Comment (alexsoble) [inline tdrs-frontend/src/components/NavItem/NavItem.jsx:17]: @hilvitzs I think the key should be unique: https://reactjs.org/docs/lists-and-keys.html

**Comment by reitermb** [comment]
👍 on internal a11y checks

**Review by carltonsmith — APPROVED** [review]
This PR is approved for Raft and submitted for QASP review to HHS

**Pull request merged**